band/models.py

```diff
@@ -91,15 +92,18 @@ def is_editor(self, member):
 
     @property
     def all_assocs(self):
-        return self.assocs.filter(member__status=MemberStatusChoices.ACTIVE)
+        return self.assocs.filter(member__status=MemberStatusChoices.ACTIVE).order_by(Lower('member__display_name'))
 
     @property
     def confirmed_assocs(self):
-        return self.assocs.filter(status=AssocStatusChoices.CONFIRMED, member__status=MemberStatusChoices.ACTIVE)
+        return self.assocs.filter(status=AssocStatusChoices.CONFIRMED, 
+                                  member__status=MemberStatusChoices.ACTIVE).order_by(Lower('member__display_name'))
 
     @property
     def confirmed_members(self):
-        return apps.get_model('member', 'Member').objects.filter(assocs__status=AssocStatusChoices.CONFIRMED, assocs__band=self, status=MemberStatusChoices.ACTIVE)
```

Comment:
Did you intend to remove the "active member" filter here? This would mean that someone who is in a band, registers a plan for a gig, and then is marked as inactive, would still show up in the planning for that gig

Reply:
Confused - didn't I just move it to the next line?

Reply:
Yes - you did. The diff formatting made it hard for me to read, but I see it now. Thanks.